plugins/modules/proxmox_kvm.py

```diff
@@ -517,6 +517,15 @@
         default: '2.0'
     type: dict
     version_added: 7.1.0
+  usb:
+    description:
+      - A hash/dictionary of usb devices for the VM. O(usb='{"key":"value", "key":"value"}').
+      - Keys allowed are - C(usb[n]) where 0 ≤ n ≤ N.
+      - Values allowed are - C(host="value|spice",mapping="value",usb3="1|0").
+      - host is either C(spice) or the USB id/port.
+      - Option C(mapping) is the mapped USB device name.
+      - Option C(usb3) enables usb3 support.
```

Comment:
Shouldn't this be
```suggestion
      - Option C(usb3) enables USB 3 support.
```
?

Reply:
it should, ive changed both